src/test/java/io/weaviate/integration/client/schema/ClientSchemaTenantOffloadingTest.java

```diff
@@ -74,7 +74,7 @@ public void shouldOffloadTenants() throws InterruptedException {
     // update tenants to FROZEN
     updateTenantStatus(className, tenants, ActivityStatus.FROZEN);
     // verify tenant status FREEZING
-    verifyEventuallyTenantStatus(className, ActivityStatus.FREEZING);
+    verifyEventuallyTenantStatus(className, ActivityStatus.FROZEN);
```

Comment:
what's motivation for change (does not match above comment)?

Reply:
this is something that you actually predicted in the last comment :) now this change from `FREEZING` to `FROZEN` happens so fast that it just hanged on this method, so I had to change it